src/editor/offset.ts

```diff
@@ -14,17 +14,17 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+import EditorModel from "./model";
+
 export default class DocumentOffset {
-    constructor(offset, atNodeEnd) {
-        this.offset = offset;
-        this.atNodeEnd = atNodeEnd;
+    constructor(public offset: number, public readonly atNodeEnd: boolean) {
```

Comment:
Are we cool with this style?

Reply:
we already use it in a few places, it is so much less chatty thankfully